cfgov/v1/models/sublanding_page.py

```diff
@@ -75,10 +75,8 @@ def get_browsefilterable_posts(self, request, limit):
                 filtered_controls.update({id: []})
             form_class = page.get_form_class()
             posts = page.get_page_set(form_class(parent=page, hostname=request.site.hostname), request.site.hostname)
-            if filtered_controls[id]:
-                filtered_controls[id] += posts
-            else:
-                filtered_controls[id] = posts
-        posts_tuple_list = [(id, post) for id, posts in filtered_controls.iteritems() for post in posts]
+            filtered_controls[id].append(posts)
+
```

Comment:
This is a good use case for [`defaultdict`](https://docs.python.org/2/library/collections.html#collections.defaultdict), which lets you avoid the check:

```py
filtered_controls = defaultdict(list)
for page in filter_pages:
    ...
    filtered_controls[id].append(posts)
```